src/sentry/snuba/metrics/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .datasource import *  # NOQA
+from .helpers import *  # NOQA
```

Comment:
Thanks for this split! We might even split `.datasource` into `.data` and `.meta`, to make it obvious which functions are involved in metadata requests and which are used for totals & time series.

Reply:
Np! I don't know if that's necessary `.data` module would just have one function that is `get_series`. I do not see the need to do this split, atleast not now